instrumentation/trace-annotation/trace-annotation.gradle

```diff
@@ -24,4 +24,5 @@ dependencies {
   testCompile(group: 'org.springframework.cloud', name: 'spring-cloud-sleuth-core', version: '+') {
     transitive = false
   }
+  testCompile group: 'io.opentelemetry', name: 'opentelemetry-contrib-auto-annotations', version: '+'
```

Comment:
Need help here. I don't understand why this dependency is needed here. All instrumentation modules depend on `agent-tooling` which depends on `agent-bootstrap` which depends on this `opentelemetry-contrib-auto-annotations`. Why this needs to be specified as test dependency? Some other tests fail because they lack this declaration. But I assumed this jar will be automatically in the classpath of tests. What don't I understand?